Resources/ios/ExampleView.js

```diff
@@ -0,0 +1,29 @@
+var UIScreen = require('UIKit/UIScreen'),
+  UIView = require("UIKit/UIView");
+
+// The Swift file we have in vendor
+var MySwiftView = require('MyFramework/MySwiftView');
+
+exports.create = function(params) {
+  params = params || {};
+
+  var view = new UIView();
+  view.setFrame(UIScreen.mainScreen().bounds);
+
+  if (params.backgroundColor) {
+
+    // Hyperloop should provide cross-platform helpers you can use anywhere in
+    // the app to do stuff like this:
+    var uiColor = Hyperloop.colorFromString(params.backgroundColor);
```

Comment:
Agree. Need to list all required built-in-components. Ticket: https://jira.appcelerator.org/browse/TIMOB-20417